src/middlewared/middlewared/client/client.py

```diff
@@ -99,10 +99,9 @@ def get_reserved_port(self):
             # iteration in the for loop
             port_low = 600
             port_high = 1024
+            size = 5
 
-            ports_to_try = random.sample(range(port_low, port_high), 5)
-
-            for port in ports_to_try:
+            for port in random_sample(port_low, port_high, size):
```

Comment:
@themylogin Yeah, I wrote this quickly on a weekend and feature creep occurred. I'll come back to this and make it so that only password/tokens related to authentication use cryptographically secure modules.